sdk/tests/integration/open_bundled_position.test.ts

```diff
@@ -603,4 +633,20 @@ describe("open_bundled_position", () => {
     });
   });
 
+  it("fail when opening a non-full range position in an infinity pool", async () => {
+    const positionBundleInfo = await initializePositionBundle(ctx, ctx.wallet.publicKey);
+    const bundleIndex = 0;
+    await assert.rejects(
+      openBundledPosition(
+        ctx,
+        whirlpoolPda.publicKey,
+        positionBundleInfo.positionBundleMintKeypair.publicKey,
+        bundleIndex,
+        tickLowerIndex,
+        tickUpperIndex
```

Comment:
Is the reason for the error is that the tick is not initializable (not multiple of tickSpacing) ?

Reply:
Good point. I think increasing the `tickUpperIndex` to 2^15 should fix this? This would also work in other cases since 2^15 is a multiple of the other tick spacings as well

Reply:
yep, I think 32768 works.